src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,230 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class CarDaoImpl implements CarDao {
+    private static final String LICENSE_NUMBER_LITERAL = "license_number";
+    private static final String NAME_LITERAL = "name";
+    private static final String CAR_ID_LITERAL = "car_id";
+    private static final String CAR_MODEL_LITERAL = "model";
+    private static final String MANUFACTURER_ID_LITERAL = "manufacturer_id";
+    private static final String MANUFACTURER_NAME_LITERAL = "manufacturer_name";
+    private static final String MANUFACTURER_COUNTRY_LITERAL = "manufacturer_country";
+
+    @Override
+    public Car create(Car car) {
+        String insertRequest = "insert into cars (model, manufacturer_id) values (?, ?)";
```

Comment:
NOTE: 
I'd write SQL query in uppercuse, so:
"INSERT INTO cars (model, manufacturer_id) VALUES (?, ?)";
I think that so is easierto read.